### DISCUSSION TITLE
forward and reverse search

### DISCUSSION THREAD

**Discussion by FuShengban** [typst/typst#861] [Q&A]
Can we support forward and reverse search like LaTeX does? This would be very useful for editing large documents.

**Comment by Enivex** [comment]
The web app already has at least some forward search functionality

https://github.com/typst/typst/issues/289

**Comment by FuShengban** [comment]
I see. But it appears that the forward search feature of the web app is not very good at supporting Chinese documents, based on my experience with a 10+ page Chinese document where the forward search result was significantly off.

**Reply by Enivex** [reply]
> I see. But it appears that the forward search feature of the web app is not very good at supporting Chinese documents, based on my experience with a 10+ page Chinese document where the forward search result was significantly off.

That sounds like a bug that should be reported.
